### ISSUE TITLE
Incorrect return type in Series.to_dict()?

### ISSUE THREAD

**Issue by janosh** [VirtusLab/pandas-stubs#83]
Shouldn't this

https://github.com/VirtusLab/pandas-stubs/blob/f521cfb5a2bda425f20d96281999791b01d4833b/third_party/3/pandas/core/series.pyi#L105

be

```py
def to_dict(self, into: Type[Mapping[K,V]] = ...) -> Type[Dict[K,V]]: ...
```

**Comment by joannasendorek** [comment]
Thanks again for your report. Will take a look

**Comment by joannasendorek** [comment]
Seems like Mapping is suggested by pandas: https://github.com/pandas-dev/pandas/blob/v1.3.2/pandas/core/series.py#L1689-L1720

**Comment by janosh** [comment]
Ah, my mistake. I find it a bit odd though that this would throw a type error on the last line

```py
from typing import Any, Dict, List, Union

import pandas as pd


dic: Dict[Union[int, str], List[Any]]

df = pd.DataFrame()

dic = df.to_dict()
```

> Incompatible types in assignment (expression has type "Union[Mapping[Union[int, str], Any], List[Any]]", variable has type "Dict[Any, Any]")mypy(error)

I mean I'm assigning the results of `to_dict()` to a `dict`...

**Comment by zkrolikowski-vl** [comment]
Fixed in 1.2.0.54
